File: web3rpc/rpc-specs/code-samples/python/kaia/gas/lowerBoundGasPrice.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.lower_bound_gas_price()

print(kaia_response)

File: web3rpc/rpc-specs/code-samples/javascript/kaia/gas/lowerBoundGasPrice.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  provider.kaia.lowerBoundGasPrice({}, (err, data, response) => {
    console.log(data);
  });
})();
